features/webdriver-bidi.yml.dist

```diff
@@ -1,4 +1,290 @@
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "126"
+  #   firefox_android: "126"
+  - webdriver.bidi.storage.deleteCookies
+  - webdriver.bidi.storage.getCookies
+  - webdriver.bidi.storage.setCookie
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "127"
+  #   firefox_android: "127"
+  - webdriver.bidi.permission
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "128"
+  #   firefox_android: "128"
+  - webdriver.bidi.permission.setPermission
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "129"
+  #   firefox_android: "129"
+  - webdriver.bidi.browser.close
+  - webdriver.bidi.browser.removeUserContext
+  - webdriver.bidi.browsingContext.close
+  - webdriver.bidi.browsingContext.handleUserPrompt
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "136"
+  #   firefox_android: "136"
+  - webdriver.bidi.session.subscribe
+  - webdriver.bidi.session.unsubscribe
+
+  # baseline: false
+  # support:
+  #   chrome: "131"
+  #   chrome_android: "131"
+  #   edge: "131"
+  #   firefox: "135"
+  #   firefox_android: "135"
+  - webdriver.bidi.browsingContext.captureScreenshot
+
+  # baseline: false
+  # support:
+  #   chrome: "132"
+  #   chrome_android: "132"
+  #   edge: "132"
+  #   firefox: "130"
+  #   firefox_android: "130"
+  - webdriver.bidi.network.setCacheBehavior
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  - webdriver.bidi.browsingContext.contextCreated_event
+  - webdriver.bidi.browsingContext.contextDestroyed_event
+  - webdriver.bidi.browsingContext.create
+  - webdriver.bidi.browsingContext.locateNodes
+  - webdriver.bidi.browsingContext.navigate
+  - webdriver.bidi.browsingContext.navigationCommitted_event
+  - webdriver.bidi.browsingContext.navigationFailed_event
+  - webdriver.bidi.browsingContext.reload
+  - webdriver.bidi.browsingContext.setViewport
+  - webdriver.bidi.browsingContext.traverseHistory
+  - webdriver.bidi.browsingContext.userPromptClosed_event
+  - webdriver.bidi.browsingContext.userPromptOpened_event
+  - webdriver.bidi.input.setFiles
+  - webdriver.bidi.log.entryAdded_event
+  - webdriver.bidi.network.continueWithAuth
+  - webdriver.bidi.script.getRealms
+  - webdriver.bidi.script.realmCreated_event
+  - webdriver.bidi.session.new
+
+  # baseline: false
+  # support:
+  #   chrome: "133"
+  #   chrome_android: "133"
+  #   edge: "133"
+  - webdriver.bidi.network.fetchError_event
+
+  # baseline: false
+  # support:
+  #   firefox: "122"
+  #   firefox_android: "122"
+  - webdriver.bidi.network.authRequired_event
+
+  # baseline: false
+  # support:
+  #   firefox: "134"
+  #   firefox_android: "134"
+  - webdriver.bidi.browser.getClientWindows
```

Comment:
These keys are supported either in chromium or in firefox, but not in both, unlike the keys that come before them.
Are they later additions? Should we ignore them in this initial bidi feature?

Reply:
It's relatively new, from https://github.com/w3c/webdriver-bidi/pull/752. I think we should just let them be later additions while pinning the overall feature to the basics.

Reply:
My inclination here would be to pin to the set of keys (or a sample of them) that show Firefox and Chrome support, since that lines up with the status we report today (though the version numbers differ). I don't think the key selection matters _that_ much until some subset is likely to become Baseline.

@captainbrosset could you set a `compute_from`? It'd be nice to merge this.